akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventsourcedRecoveringEvents.scala

```diff
@@ -175,8 +171,8 @@ private[persistence] class EventsourcedRecoveringEvents[C, E, S](override val se
     cancelRecoveryTimer(setup.timers)
   }
 
-  // protect against snapshot stalling forever because of journal overloaded and such
-  private val RecoveryTickTimerKey = "recovery-tick"
+  // protect against event recovery stalling forever because of journal overloaded and such
+  private val RecoveryTickTimerKey = "event-recovery-tick"
   private def startRecoveryTimer(timers: TimerScheduler[InternalProtocol], timeout: FiniteDuration): Unit =
     timers.startPeriodicTimer(RecoveryTickTimerKey, RecoveryTickEvent(snapshot = false), timeout)
```

Comment:
`snapshot = true`, but I don't think the flag is needed at all

Reply:
Yeah with the timer cancelations I see, I'll keep it for now though since timer semantics may change -- see Johan's PR